app/views/channelcomparison.php

```diff
@@ -58,25 +58,25 @@
 <h3 id="new_strings">No new strings have been added</h3>
 
 <?php else : ?>
-<h3 id="new_strings">New added strings in <?=$locale?> between <?=$repos_nice_names[$chan1]?> and <?=$repos_nice_names[$chan2]?></h3>
+<h3 id="new_strings">New strings added in <em><?=$locale?></em> between <?=$repos_nice_names[$chan1]?> and <?=$repos_nice_names[$chan2]?></h3>
 <table class="collapsable">
     <thead>
-        <tr>
-            <th>Key</th>
-            <th>Value</th>
-            <th>en-US value</th>
+        <tr class='column_headers'>
+            <th>Entity</th>
+            <th>en-US</th>
+            <th><?=$locale?></th>
         </tr>
     </thead>
     <tbody>
-    <?php foreach ($new_strings as $key => $value): ?>
+    <?php foreach ($new_strings as $string_id => $string_values): ?>
     <tr>
-        <td><span class="celltitle">Key</span><div class="string"><?=showResults::formatEntity($key)?></div></td>
-        <td><span class="celltitle">Value</span><div class="string"><?=Utils::secureText($value)?></div></td>
-        <?php if (isset($new_en_US_strings[$key])): ?>
-        <td><span class="celltitle">en-US Value</span><div class="string"><?=Utils::secureText($new_en_US_strings[$key])?></div></td>
+        <td><span class="celltitle">Entity</span><div class="string"><?=ShowResults::formatEntity($string_id)?></div></td>
+        <?php if ($new_strings[$string_id]['reference'] != '@N/A@'): ?>
+        <td><span class="celltitle">en-US</span><div class="string"><?=Utils::secureText($string_values['reference'])?></div></td>
         <?php else: ?>
-        <td><span class='celltitle'>en-US value</span><div class="string">(not available)</div></td>
+        <td><span class='celltitle'>en-US</span><div class="string"><em class="error">(not available)</em></div></td>
         <?php endif; ?>
+        <td><span class="celltitle"><?=$locale?></span><div class="string"><?=Utils::secureText($string_values['translation'])?></div></td>
```

Comment:
Since we always have the same formatting for the td cell and it's quite verbose, I am wondering if we couldn't do that:
```php
namespace Transvision;

// Helper anonymous variable to output a formatted table cell
$td = function ($key, $value) {
    return "<td><span class=\"celltitle\">{$key}</span>"
           . "<div class=²\"string\">{$value}</div></td>";
};
?>
…
    <?php foreach ($common_strings as $key => $value) : ?>
    <tr>
        <?=$td('Key', ShowResults::formatEntity($key))?>
        <?=$td($chan1, Utils::secureText($value))?>
        <?=$td($chan2, Utils::secureText($strings[$chan2][$key]))?>
    </tr>
    <?php endforeach; ?>
…
    <tr>
        <?=$td('Entity', ShowResults::formatEntity($string_id))?>
        <?php if ($new_strings[$string_id]['reference'] != '@N/A@'): ?>
        <?=$td('en-US', Utils::secureText($string_values['reference']))?>
        <?php else: ?>
        <?=$td('en-US', '<em class="error">(not available)</em>')?>
        <?php endif; ?>
        <?=$td($locale, Utils::secureText($string_values['translation']))?>
    </tr>
```